pkg/core/common/docker/containerlist.go

```diff
@@ -117,33 +172,9 @@ func ListAndWatchContainers(ctx context.Context, client *docker.Client, changeHa
 
 	<-watchStarted
 
-	f := filters.NewArgs()
-	f.Add("status", "running")
-	options := dtypes.ContainerListOptions{
-		Filters: f,
-	}
-	containerList, err := client.ContainerList(ctx, options)
+	err := syncContainerList()
```

Comment:
Move this to immediately before the for loop on line 122 and then always return nil err from this method.  We don't want the monitor to crash and not work if docker happens to be down when the agent starts up.

Reply:
Then you can get rid of the watchStarted stuff as well.

Reply:
done. 
also did more cleanup for unused